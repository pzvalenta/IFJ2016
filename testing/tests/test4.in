class Main
{
  static void run()
  {
    int a;
    ifj16.print("Zad");
    a = ifj16.readInt();
    int vysl;
    if (a < 0) { 
      ifj16.print("Fak!\n");
  }
    else {
      vysl = 1;
      while (a > 0) {
      vysl = vysl * a;
      a = a - 1;
      }
      ifj16.print("Vysledek je: " + vysl + "\n");
    }
    
                boolean b1 = true;
                boolean b2 = false;
    
    for(int var =0; var < 5 ; var++)
{
        ifj2016.print(“Var is : “ + var);
 
        if(var == 3)
                break;
}        
          String str1;
          str1 = "Toto je nejaky text";
         double d=10.10;
             int k,l,m,n,p;
             k=0; 
             l=1; 
             m=2; 
             n=3; 
             p=4; 
   
             k=k/l;
          
             if(m >= 3)
                break; 
            if(n <= 3)
                break;  
               if(p != 3)
                continue; 
                  
            return a;
    }
    
}