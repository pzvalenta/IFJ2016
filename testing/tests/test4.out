NO	ID
1	23	N/A
2	11	node->name->data: 'Main'
3	103	N/A
4	34	N/A
5	36	N/A
6	11	node->name->data: 'run'
7	101	N/A
8	102	N/A
9	103	N/A
10	31	N/A
11	11	node->name->data: 'a'
12	107	N/A
13	11	node->name->data: 'ifj16'
14	109	N/A
15	11	node->name->data: 'print'
16	101	N/A
17	43	token->data.s :"Zad"
18	102	N/A
19	107	N/A
20	11	node->name->data: 'a'
21	51	N/A
22	11	node->name->data: 'ifj16'
23	109	N/A
24	11	node->name->data: 'readInt'
25	101	N/A
26	102	N/A
27	31	N/A
28	11	node->name->data: 'vysl'
29	107	N/A
30	30	N/A
31	101	N/A
32	11	node->name->data: 'a'
33	55	N/A
34	43	token->data.s :"(null)"
35	102	N/A
36	103	N/A
37	11	node->name->data: 'ifj16'
38	109	N/A
39	11	node->name->data: 'print'
40	101	N/A
41	43	token->data.s: "Fak!\n"
42	102	N/A
43	107	N/A
44	104	N/A
45	27	N/A
46	103	N/A
47	11	node->name->data: 'vysl'
48	51	N/A
49	41	token->data.i :'1'
50	107	N/A
51	37	N/A
52	101	N/A
53	1	node->name->data: 'a'
54	55	N/A
55	43	token->data.s :"(null)"
56	102	N/A
57	103	N/A
58	11	node->name->data: 'vysl'
59	51	N/A
60	11	node->name->data: 'vysl'
61	61	N/A
62	11	node->name->data: 'a'
63	107	N/A
64	11	node->name->data: 'a'
65	51	N/A
66	11	node->name->data: 'a'
67	57	N/A
68	107	N/A
69	104	N/A
70	11	node->name->data: 'ifj16'
71	109	N/A
72	11	node->name->data: 'print'
73	101	N/A
74	43	token->data.s :"Vysledek je: "
75	53	N/A
76	11	node->name->data: 'vysl'
77	53	N/A
78	43	token->data.s :"\n"
79	102	N/A
80	107	N/A
81	104	N/A
82	21	N/A
83	11	node->name->data: 'b1'
84	51	N/A
85	35	N/A
86	107	N/A
87	21	N/A
88	11	node->name->data: 'b2'
89	51	N/A
90	28	N/A
91	107	N/A
92	29	N/A
93	101	N/A
94	31	N/A
95	11	node->name->data: 'var'
96	51	N/A
97	43	token->data.s :"(null)"
98	107	N/A
99	11	node->name->data: 'var'
100	59	N/A
101	41	token->data.i :'5'
102	107	N/A
103	11	node->name->data: 'var'
104	53	N/A
105	53	N/A
106	102	N/A
107	103	N/A
108	11	node->name->data: 'ifj16'
109	109	N/A
110	11	node->name->data: 'print'
111	101	N/A
112	43	token->data.s :"Var is : "
113	53	N/A
114	11	node->name->data: 'var'
115	102	N/A
116	107	N/A
117	30	N/A
118	101	N/A
119	11	node->name->data: 'var'
120	52	N/A
121	41	token->data.i :'3'
122	22	N/A
123	107	N/A
124	104	N/A
125	33	N/A
126	11	node->name->data: 'str1'
127	107	N/A
128	11	node->name->data: 'str1'
129	51	N/A
130	43	token->data.s :"Toto je nejaky text"
131	107	N/A
132	26	N/A
133	11	node->name->data: 'd'
134	51	N/A
135	42	token->data.f :'10.10'
136	107	N/A
137	31	N/A
138	11	node->name->data: 'k'
139	108	N/A
140	11	node->name->data: 'l'
141	108	N/A
142	11	node->name->data: 'm'
143	108	N/A
144	11	node->name->data: 'n'
145	108	N/A
146	11	node->name->data: 'p'
147	107	N/A
148	11	node->name->data: 'k'
149	51	N/A
150	43	token->data.s :"(null)"
151	107	N/A
152	11	node->name->data: 'l'
153	51	N/A
154	41	token->data.i :'1'
155	107	N/A
156	11	node->name->data: 'm'
157	51	N/A
158	41	token->data.i :'2'
159	107	N/A
160	11	node->name->data: 'n'
161	51	N/A
162	41	token->data.i :'3'
163	107	N/A
164	11	node->name->data: 'p'
165	51	N/A
166	41	token->data.i :'4'
167	107	N/A
168	11	node->name->data: 'k'
169	51	N/A
170	11	node->name->data: 'k'
171	65	N/A 
172	11	node->name->data: 'l'
173	107	N/A
174	30	N/A
175	101	N/A
176	11	node->name->data: 'm'
177	56	N/A
178	41	token->data.i :'3'
179	102	N/A
180	22	N/A
181	107	N/A
182	30	N/A
183	101	N/A
184	11	node->name->data: 'n'
185	60	N/A
186	41	token->data.i :'3'
187	102	N/A
188	22	N/A
189	107	N/A
190	30	N/A
191	101	N/A
192	11	node->name->data: 'p'
193	64	N/A
194	41	token->data.i :'3'
195	102	N/A
196	24	N/A
197	107	N/A
198	32	N/A
199	11	node->name->data: 'a'
200	107	N/A
201	104	N/A
202	104	N/A